unattended_installer/install_functions/installCommon.sh

```diff
@@ -600,3 +600,57 @@ function installCommon_startService() {
     fi
 
 }
+
+function installCommon_setupFirewall() {
+
+    if [ "$#" -ne 1 ]; then
+        common_logger -e "installCommon_setupFirewall must be called with 1 argument."
+        exit 1
+    fi
+
+    FIREWALLD_EXISTS=0
+    IPTABLES_EXISTS=0
+
+    if [ -x "$(command -v firewalld)" ]; then
+        FIREWALLD_EXISTS=1
+    fi
+
+    if [ -x "$(command -v iptables)" ]; then
+        IPTABLES_EXISTS=1
+    fi
+
+    if [[ ${FIREWALLD_EXISTS} || ${IPTABLES_EXISTS} ]]; then
+
+        common_logger "Setting up firewall rules for service ${1}."
+
+        if [ "${1}" == 'wazuh-manager' ]; then
+            ports=("1514" "1515" "1516" "55000")
+        elif [ "${1}" == 'wazuh-indexer' ]; then
+            ports=("9200" "9300-9400")
+        elif [ "${1}" == 'wazuh-dashboard' ]; then
+            ports=("443")
+        fi
+
+        for port_number in "${ports[@]}"; do
+            common_logger -d "Enabling port ${port_number} on the firewall."
+            if [ "${FIREWALLD_EXISTS}" ]; then
+                eval "sudo firewall-cmd --permanent --add-port=${port_number}/tcp ${debug}"
+            elif [ "${IPTABLES_EXISTS}" ]; then
+                eval "sudo iptables -I INPUT -p tcp -m tcp --dport ${port_number} -j ACCEPT ${debug}"
+            fi
+        done
+
+        if [ "${FIREWALLD_EXISTS}" ]; then
+            eval "sudo firewall-cmd --reload ${debug}"
+
+        elif [ "${IPTABLES_EXISTS}" ]; then
+            eval "sudo service iptables save ${debug}"
+        fi
+        common_logger -d "Firewall rules saved/reloaded."
+        common_logger "Successfully set up basic firewall rules for service ${1}."
+        common_logger "It is suggested to verify the rules on the host firewall in accordance with the Wazuh documentation."
+    fi
+
+    common_logger -e "Could not find a firewall. Please configure ports on the host firewall in accordance with the Wazuh documentation."
+    return 1
```

Comment:
This function always return 1 (as error)

Reply:
Agree. I missed adding return 0 to the success conditions.